kombu/asynchronous/aws/sqs/connection.py

```diff
@@ -25,6 +29,75 @@ def __init__(self, sqs_connection, debug=0, region=None, **kwargs):
             **kwargs
         )
 
+    def _create_query_request(self, operation, params, queue_url, verb, callback=None):  # noqa
+        params = params.copy()
+        if operation:
+            params['Action'] = operation
+
+        # defaults for non-get
+        param_payload = {'data': params}
+        if verb.lower() == 'get':
+            # query-based opts
+            param_payload = {'params': params}
+
+        return AWSRequest(method=verb, url=queue_url, **param_payload)
```

Comment:
Can we keep `verb` consistent with `method` used elsewhere?

Reply:
@dhegberg , I tried to stay consistent with the original implementation to reduce the amount of changes. Having said that, yeah, it is confusing, so I will just rename everything to method.